projects/angular-showcase/src/app/features/business-component-viewer/business-component-viewer/business-component-viewer.component.scss

```diff
@@ -0,0 +1,3 @@
+::ng-deep pre {
```

Comment:
I think it would be better to do this globally in the showcase styles.scss
Or was there a reason why you wanted to avoid that?
e.g.
```scss
sbb-business {
  sbb-markdown-viewer,
  sbb-component-viewer {
    pre {
      ...
    }

    sbb-example-viewer pre {
      ...
    }
  }
}
```

Reply:
I did this because there is a different negative margin in business than the public showcase. What we could do is setting different tags for public and business (e.g. sbb-public-component-viewer, sbb-business-component-viewer, ..., since the tag names are otherwise irrelevant) and then configure it globally in the styles.scss. I guess that would be a cleaner solution than having multiple files and using the deprecated ng-deep.

> I'm starting to question, whether the business part of the showcase should use the business tabs component. I wanted to directly show how the business section looks, but it seems to break the design.
Do you see an easy solution to use the public tabs for the showcase and then still include the business tabs for the examples?

Reply:
Actually nevermind the first solution approach, the tag name is used.

Reply:
Sorry, I'm a bit confused now. 😅 
Are you going to change the styling approach or would you like to keep it as ::ng-deep?

> Do you see an easy/clean solution to use the public tabs for the showcase and then still include the business tabs for the examples?

I think the way you now implemented it looks good, so I don't have a complaint anymore.

Reply:
I added a class to the different example viewers and styled the margins globally.